define/build.go

```diff
@@ -136,6 +137,12 @@ type BuildOptions struct {
 	RuntimeArgs []string
 	// TransientMounts is a list of mounts that won't be kept in the image.
 	TransientMounts []string
+	// CacheFrom specifies any remote repository which can be treated as
+	// potential cache source.
+	CacheFrom *reference.Named
+	// CacheTo specifies any remote repository which can be treated as
+	// potential cache destination.
+	CacheTo *reference.Named
```

Comment:
Go interface values (sadly) include a `nil` value, so this doesn’t need the `*`. And not including it would be better because the corner case of “non-`nil` pointer to `nil`” would go away.

Reply:
Removed the reference and passing the actual struct now.